.azuredevops/deploy-platform-core.yaml

```diff
@@ -99,6 +99,11 @@ extends:
             privateEndpointDnsRecordsForResources:
               - resourceName: $(infraResourceNamePrefix)$(nc_resource_servicebus)$(nc_instance_regionid)01
                 resourceGroupName: $(servicesResourceGroup)
+      - name: front_door
```

Comment:
Do we really need a separate job for just the front door?
Should this be added to the network job? @ppotturi @DaRoszko

Reply:
As discussed, we will revisit this later when other components are deployed from the same pipeline.